src/Altinn.AccessManagement/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Altinn.AccessManagement.Core.Clients.Interfaces;
+using Altinn.AccessManagement.Core.Configuration;
+using Altinn.AccessManagement.Integration.Configuration;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.AccessManagement.Controllers
+{
+    /// <summary>
+    /// Exposes API endpoints related to authentication.
+    /// </summary>
+    [Route("accessmanagement/")]
+    [ApiController]
+    public class AuthenticationController : ControllerBase
+    {
+        private readonly IAuthenticationClient _authenticationClient;
+        private readonly GeneralSettings _settings;
+        private readonly ILogger _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuthenticationController"/> class
+        /// </summary>
+        public AuthenticationController(
+            IAuthenticationClient authenticationClient, 
+            IOptions<GeneralSettings> settings, 
+            IOptions<PlatformSettings> platformSettings,
```

Comment:
platformSettings option is unused. Some overlap of value between general and platform settings which should be cleaned up?
e.g. PlatformSettings.JwtCookieName and GeneralSettings.RuntimeCookieName seems to be the same